android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayout.java

```diff
@@ -279,7 +279,7 @@ public void onClick(View v) {
 
   @Override
   public boolean onLongClick(View v) {
-      String description = "";
+      String description = null;
```

Comment:
According to https://source.chromium.org/chromium/chromium/src/+/master:ui/android/java/src/org/chromium/ui/widget/Toast.java;l=182?q=showAnchoredToast&ss=chromium&originalUrl=https:%2F%2Fcs.chromium.org%2F
`Toast.showAnchoredToast` would return false if description is null.

Reply:
I think this would be useful as a comment in the code

Reply:
Oops, merged too fast without seeing this, I have another WIP PR which would change this file too, I'll add a comment here in that PR.

Reply:
Added in https://github.com/brave/brave-core/pull/4617/commits/01633c156a0b223af72536e8366c07d68b1ef439#diff-a6c2c460ae34706b2d235a605f16caa3R313-R314